src/charm.py

```diff
@@ -272,6 +404,16 @@ def _admin_ui_pebble_layer(self) -> Layer:
             container_env["OTEL_HTTP_ENDPOINT"] = self._tracing_endpoint_info_http
             container_env["OTEL_GRPC_ENDPOINT"] = self._tracing_endpoint_info_grpc
 
+        if (openfga_info := self._get_openfga_store_info()) and (
+            model_id := self._openfga_model_id
+        ):
+            container_env["AUTHORIZATION_ENABLED"] = True
+            container_env["OPENFGA_AUTHORIZATION_MODEL_ID"] = model_id
+            container_env["OPENFGA_STORE_ID"] = openfga_info.store_id
+            container_env["OPENFGA_API_TOKEN"] = openfga_info.token
+            container_env["OPENFGA_API_SCHEME"] = "http"
+            container_env["OPENFGA_API_HOST"] = openfga_info.http_api_url.strip("http://")
```

Comment:
Let's not hardcode the `http` scheme. We are going to add https support to the openfga charm. I think we should parse the URL and use that to configure the app.